tests/test_lsst.rs

```diff
@@ -0,0 +1,79 @@
+use boom::alert::{AlertWorker, LsstAlertWorker};
+
+const CONFIG_FILE: &str = "tests/config.test.yaml";
+
+#[tokio::test]
+async fn test_lsst_alert_from_avro_bytes() {
+    let mut alert_worker = LsstAlertWorker::new(CONFIG_FILE).await.unwrap();
+
+    let file_name = "tests/data/alerts/lsst/0.avro";
+    let bytes_content = std::fs::read(file_name).unwrap();
+    let mut alert = alert_worker
+        .alert_from_avro_bytes(&bytes_content)
+        .await
+        .unwrap();
+
+    assert_eq!(alert.candid, 25409136044802067);
+    assert_eq!(alert.candidate.object_id.unwrap(), 25401295582003262);
+
+    assert!((alert.candidate.ra - 149.802106).abs() < 1e-6);
```

Comment:
This work, but if you're interested, there are various crates available that provide assertion macros to compare floats. Totally up to you.